packages/web-components/test/walletConnection.test.js

```diff
@@ -52,6 +52,7 @@ describe('makeAgoricWalletConnection', () => {
 
     // Don't bother faking keplr, just expect it to try to enable and fail.
     await expect(() =>
+      // @ts-expect-error fake partial watcher implementation
```

Comment:
you could cast the `watcher` so you don't have to ignore at each site. up to you

Reply:
Casting requires stubbing out more of the functions than the test uses, or else I'm getting type errors. Unless you mean casting to `any`, but I think the expect-error with explanation would be better.

Reply:
I meant to put the one `ts-expect-error` at the cast site. then it will let you lie about what the type is.

i agree that any expect-error should have an explanation